package-scripts.js

```diff
@@ -43,63 +53,58 @@ module.exports = {
       automated:
         "mocha --require babel-register --require chai --timeout 15000",
       watch: 'nps "test.automated --watch"',
-      debug: 'nps "test.automated --debug"',
       bail: 'nps "test.automated --bail"',
-      testrpc: {
+      ganache: {
         /** the same command as is used by default by daostack-arc */
-        run: `testrpc -l ${config.get('gasLimit')} --account="0x0191ecbd1b150b8a3c27c27010ba51b45521689611e669109e034fd66ae69621,9999999999999999999999999999999999999999999" --account="0x00f360233e89c65970a41d4a85990ec6669526b2230e867c352130151453180d,9999999999999999999999999999999999999999999" --account="0x987a26abca7432016104ce2f24ce639340e25afe06ac69f68791399e7a5d1028,9999999999999999999999999999999999999999999" --account="0x89af34b1b7347834048b99423dad174a14bf14540d720d72c16ae92e94b987cb,9999999999999999999999999999999999999999999" --account="0xc867be647eb2bc51e4c0d61066859875cf3634fe949b6f5f85c69ab90e485b37,9999999999999999999999999999999999999999999" --account="0xefabcc2377dee5e51b5a9e65a3854aec85fbbec3cb584d8ad4f9679869fb33c6,9999999999999999999999999999999999999999999"`
+        run: series(
+          `cd ${pathDaostackArcRepo}`,
+          "npm run ganache",
+          `cd ${pathArcJsRoot}`
+        ),
+        runAsync: runInNewWindow("npm start test.ganache.run")
       },
       testrpcDb: {
         /**
-         * Full workflow to create custom contracts.  This can take a long time as there may
-         * be thousands of files to delete.
+         * testrpcDB scripts are handy for doing development against ganache, enabling you to
```

Comment:
Why it is needed ?
Deployment to testrpc/ganache can be done at no time without cost.